javascript/src/main/pages/AnalyzeMessageData/AnalyzeReactions.js

```diff
@@ -0,0 +1,52 @@
+import React, {useState} from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+import useSWR from "swr";
+import { fetchWithToken } from "main/utils/fetch";
+import MessageTableReaction from "main/components/ChannelMessageReaction/MessageTableReaction"
+import { Form, Button } from "react-bootstrap";
+
+
+const AnalyzeReactions = () => {
+    const { getAccessTokenSilently: getToken } = useAuth0();
+    const [ searchReaction, setSearchReaction ] = useState('');
+    const [ searchResults, setSearchResults ] = useState([]);
+    
+    const handleSearchReactionOnChange = (event) => {
+        setSearchReaction(event.target.value);
+    };
+    
+    const handleSearchReactionOnSubmit = () => {
+        var url = '';
+        if(searchReaction === "+1") {
+            const searchReaction2 = "%2B1";
+            const url1 = `/api/members/messages/reactionsearch?searchReaction=${searchReaction2}`;
+            url = url1;
+        } else {
+            const url2 = `/api/members/messages/reactionsearch?searchReaction=${searchReaction}`;
+            url = url2;
+        }
+        const options = {
+            method: 'GET',
+        }
+        fetchWithToken(url, getToken, options)
+            .then((response) => {
+                setSearchResults(response);
+            })
+    };
+    return ( 
+        <>
+            <h1> Search Results </h1>
+            <Form>
+                <Form.Group controlId="searchReaction" onChange={handleSearchReactionOnChange}>
```

Comment:
Change onChange to onSubmit and the type="submit" for the Search button

Reply:
Another comment that was marked as "resolved" with no actual resolution. Can you please address this?